web/src/people.ts

```diff
@@ -660,6 +662,28 @@ export function pm_with_operand_ids(operand: string): number[] | undefined {
     return user_ids;
 }
 
+export function filter_guest_emails(email_string: string): string[] | undefined {
+    const emails = email_string.split(",");
+    const guest_emails = emails.filter((email) => get_by_email(email)?.is_guest);
+    if (guest_emails.length === 0) {
+        return undefined;
+    }
+    return guest_emails;
+}
+
+export function get_dm_guest_banner_text(guest_emails: string[]): string {
+    const names = emails_to_full_names_string(guest_emails).split(", ");
+    if (names.length === 1) {
+        return $t({defaultMessage: "{name} is a guest in this organization."}, {name: names[0]});
+    }
+    const banner_names = names.slice(0, -1).join(", ");
+    const last_name = names.at(-1);
```

Comment:
last_name might be a confusing name, where it gets confused with the last name of someone's name. Could we name this something different? perhaps `last_guest_full_name`.